src/pkg/services/m365/api/lists_pager.go

```diff
@@ -77,6 +78,26 @@ func (c Lists) GetLists(
 	return items, graph.Stack(ctx, err).OrNil()
 }
 
+func (c Lists) GetListsByCollisionKey(
+	ctx context.Context,
+	siteID string,
+) (map[string]string, error) {
+	pager := c.NewListsPager(siteID, CallConfig{Select: listCollisionKeyProps()})
+
+	items, err := pagers.BatchEnumerateItems[models.Listable](ctx, pager)
+	if err != nil {
+		return nil, graph.Wrap(ctx, err, "enumerating lists")
```

Comment:
```suggestion
		return nil, clues.WrapWC(ctx, err, "enumerating lists")
```

We are moving away from `graph.Wrap`, see https://github.com/alcionai/corso/pull/5018/files